tests/tck-build-logic/src/main/groovy/org.graalvm.internal.tck.gradle

```diff
@@ -70,6 +74,9 @@ graalvmNative {
     binaries {
         test {
             configurationFileDirectories.from(metadataFile)
+            if (override) {
+                excludeConfig.put(libraryGAV, ["*"])
```

Comment:
@lazar-mitrovic Not sure if `"*"` is expected or if it should be `".*"` to be a valid regexp.